rust/read_json_file.rs

```diff
@@ -0,0 +1,46 @@
+//cargo-deps: json="0.12.4"
+
+use std::fs::File;
+use std::io::Read;
+use std::env;
+use json::{parse, JsonValue};
+
+
+extern crate json;
+
+#[derive(Debug)]
+struct Person {
+    age: u32,
+    first_name: String,
+}
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
```suggestion
fn main() {
```

Let's not use the `Result<...>` + `?;` approach, and instead use `.unwrap();`